### PR TITLE
Added test descriptions for documentation.

### PR DESCRIPTION
**Pull request by MiguelazoDS**
|Related issue|
|---|
|#2154|

## Description

This PR includes new documentation blocks for force_options tests and improves already documented tests

## Tests

- [X] Python codebase satisfies PEP-8 style style guide. `pycodestyle --max-line-length=120 --show-source --show-pep8 file.py`.
- [X] Python codebase is documented following the Google Style for Python docstrings.

### CODE REVIEW

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Just some minor fixes to do.

File: tests/integration/test_authd/test_authd.py [hunk 1]
```diff
@@ -108,16 +108,15 @@ def test_ossec_auth_messages(get_configuration, set_up_groups, configure_environ
                              clean_client_keys_file_module, restart_authd, wait_for_authd_startup_module,
                              connect_to_sockets_module):
     '''
-    description: Check if when the `wazuh-authd` daemon receives different kinds of enrollment requests,
-                 it responds appropriately to them. In this case, the enrollment requests
-                 are sent to an IP v4 network socket.
+    description:
+        Checks if when the `wazuh-authd` daemon receives different kinds of enrollment requests,
```
[the remark this answers is not in the document]
Comment (MiguelazoDS) [reply tests/integration/test_authd/test_authd.py]: Good observation, I overlooked. Changed!

File: tests/integration/test_authd/test_authd_agents_ctx.py [hunk 2]
```diff
@@ -394,12 +394,14 @@ def duplicate_name_agent_delete_test(server):
 def test_ossec_authd_agents_ctx_main(get_configuration, set_up_groups, configure_environment,
                                      configure_sockets_environment, connect_to_sockets_module):
     '''
-    description: Check if when the 'wazuh-authd' daemon receives an enrollment request from an agent
-                 that has an IP address or name that is already registered, 'authd' creates a record
-                 for the new agent and deletes the old one. In this case, the enrollment requests
-                 are sent to an IP v4 network socket.
+    description:
+        Check if when the 'wazuh-authd' daemon receives an enrollment request from an agent
```
[the remark this answers is not in the document]
Comment (MiguelazoDS) [reply tests/integration/test_authd/test_authd_agents_ctx.py:398]: Fixed. I found the same typo in other files too.

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**